Composer/packages/client/src/pages/language-understanding/table-view.tsx

```diff
@@ -148,19 +148,23 @@ const TableView: React.FC<TableViewProps> = props => {
           );
         },
       },
-      {
-        key: 'beenUsed',
-        name: formatMessage('Been used'),
-        fieldName: 'beenUsed',
-        minWidth: 100,
-        maxWidth: 100,
-        isResizable: true,
-        isCollapsable: true,
-        data: 'string',
-        onRender: item => {
-          return item.used ? <IconButton iconProps={{ iconName: 'Accept' }} /> : <div />;
-        },
-      },
+      // {
+      //   key: 'beenUsed',
+      //   name: formatMessage('Been used'),
+      //   fieldName: 'beenUsed',
+      //   minWidth: 100,
+      //   maxWidth: 100,
+      //   isResizable: true,
+      //   isCollapsable: true,
+      //   data: 'string',
+      //   onRender: item => {
+      //     return item.used ? (
+      //       <FontIcon iconName="Accept" aria-label={formatMessage('Used')} className={iconClass} />
+      //     ) : (
+      //       <div />
+      //     );
+      //   },
+      // },
```

Comment:
Delete uncommented code

Reply:
I think the intention is to bring this back after the bug is fixed.

Reply:
Yes, when we can explicitly calculate a lu intent is used or not.